dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -37,49 +39,90 @@ public class TextMemorySkill
     /// </summary>
     public const string KeyParam = "key";
 
+    /// <summary>
+    /// Name of the context variable used to specify the number of memories to recall
+    /// </summary>
+    public const string LimitParam = "limit";
+
     private const string DefaultCollection = "generic";
     private const string DefaultRelevance = "0.75";
+    private const string DefaultLimit = "1";
+
+    /// <summary>
+    /// Recall a specific memory
+    /// </summary>
+    /// <example>
+    /// SKContext[TextMemorySkill.KeyParam] = "countryInfo1"
+    /// {{memory.recall_specific }}
+    /// </example>
+    /// <param name="context">Contains the 'collection' containing the memory to recall and the `key` associated with it.</param>
+    [SKFunction("Recall a specific memory")]
+    [SKFunctionName("Recall_Specific")]
```

Comment:
Random: I feel like we should start exposing these values as constants like with params so it's easier to reference via the ISKFunction dictionary.